pkg/agent/openflow/client.go

```diff
@@ -799,6 +799,9 @@ func (c *client) InstallServiceFlows(config *types.ServiceConfig) error {
 	if config.IsDSR {
 		flows = append(flows, c.featureService.dsrServiceMarkFlow(config))
 	}
+	if len(config.LoadBalancerSourceRanges) != 0 {
```

Comment:
ditto, does it need to be scope with condition `proxyLoadBalancerIPs==true`?

Reply:
No, the function will be call in pkg/agent/proxier.go when `proxyLoadBalancerIPs==true`.